CMakeLists.txt

```diff
@@ -568,16 +568,8 @@ MESSAGE(
 INCLUDE(InstallRequiredSystemLibraries)
 SET(CPACK_PACKAGE_DESCRIPTION_SUMMARY "LMMS - easy music production for everyone!")
 SET(CPACK_PACKAGE_VENDOR "LMMS Developers")
-IF(LMMS_BUILD_APPLE)
-	CONFIGURE_FILE("${CMAKE_SOURCE_DIR}/README" "${CMAKE_BINARY_DIR}/README.txt" COPYONLY)
-	CONFIGURE_FILE("${CMAKE_SOURCE_DIR}/COPYING" "${CMAKE_BINARY_DIR}/COPYING.txt" COPYONLY)
-	SET(CPACK_PACKAGE_DESCRIPTION_FILE "${CMAKE_BINARY_DIR}/README.txt")
-	SET(CPACK_RESOURCE_FILE_LICENSE "${CMAKE_BINARY_DIR}/COPYING.txt")
-ELSE(LMMS_BUILD_APPLE)
-	SET(CPACK_PACKAGE_DESCRIPTION_FILE "${CMAKE_SOURCE_DIR}/README")
-	SET(CPACK_RESOURCE_FILE_LICENSE "${CMAKE_SOURCE_DIR}/COPYING")
-ENDIF(LMMS_BUILD_APPLE)
-
```

Comment:
This mess seems to be an Apple packaging preference to use a `foo.TXT` file rather than a `foo` file.  Regardless, we don't let cmake do our apple packaging currently, so its not needed any longer.  If we switch back to cmake for Apple, we can revisit this section then.